frontend/src/Screens/GlobalComponents/Navbar/Navbar_Active.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './Navbar.css';
+import '../../../Style/Colors.css'
+import ActiveMode from './Components/ActiveMode/ActiveMode';
+
+function Navbar() {
+    return (
+        <div>
+            <div className='navbar'>
+                <div className='navbar__activemode_leftSide'>
+                    <ActiveMode />
+                    <h2 className='about'>About</h2>
+                </div>
+                <div className='navbar__activemode_rightSide'>
+                    <h2 className='logo'>LOGO</h2>
```

Comment:
this is sould be inside the Active and the Default Mode component

Reply:
Please explain